src/directives/my_identity.ts

```diff
@@ -55,9 +55,8 @@ export default [
             }],
             template: `
                 <div class="my-threema-information" ng-click="ctrl.showQRCode()">
-                    <div class="threema-id" ng-cloak>
-                        <span translate>messenger.MY_PUBLIC_NICKNAME</span>
-                        {{ctrl.identity.publicNickname}}
+                    <div class="nickname" ng-cloak
+                        ng-bind-html="ctrl.identity.publicNickname | emojify | emptyToPlaceholder: '-'">
```

Comment:
Shouldn't we use the identity as fallback, instead of `-`?

Reply:
no, it should be not possible to set a blank nickname in the app :-)